index.js

```diff
@@ -1231,7 +1233,25 @@ class Command extends EventEmitter {
 
     if (alias === command._name) throw new Error('Command alias can\'t be the same as its name');
 
-    command._alias = alias;
+    command._aliases.push(alias);
+    return this;
+  };
+
+  /**
+   * Set aliases for the command.
+   *
+   * Only the first alias is shown in the auto-generated help.
+   *
+   * @param {string[]} [aliases]
+   * @return {string[]|Command}
+   * @api public
+   */
+
+  aliases(aliases) {
+    // Getter for the array of aliases is the main reason for having aliases() in addition to alias().
+    if (aliases === undefined) return this._aliases;
+
```

Comment:
What do you think about this?
```suggestion
    if (arguments.length > 1) aliases = Array.from(arguments);
```

Reply:
Fair question. Using `arguments.length` means some smart editors list multiple arguments in the auto-detected signature. I actually removed all uses just recently! #1235

An alternative would to use separate arguments _instead_ of array. I prefer array in this case for the symmetry with the getter/setter signatures.

Reply:
Oh ok, I don't even know what would be the ts definition for that.
Array makes more sense from a typing pov, and varargs is just a preferred lazy typing person.